modules/common/src/bugsnag/breadcrumbs.ts

```diff
@@ -0,0 +1,24 @@
+import * as httpContext from 'express-http-context';
+import {REQUEST_UNIQUE_ID_BREADCRUMBS_KEY} from '../constants';
+
+export function getBreadcrumbs (): Record<string, any> {
+    return httpContext.get(REQUEST_UNIQUE_ID_BREADCRUMBS_KEY) || {};
+}
+
+export function leaveBreadcrumb (name: string, metaData?: object): void {
+    try {
+        // currently on 12.4.2019 are breadcrumbs not yet supported in NodeJS, but we are prepared, and currently fall backing to notify
+        // bugsnagClient.leaveBreadcrumb(name, metaData);
```

Comment:
Right, they now support breadcrumbs a bit better - I can assign them to the notified event, and they provide a proper type for them. I adjusted this code, but I noticed that we are never using `leaveBreadcrumb` anyway, so it will always be an empty array :/

Reply:
I think it's fine. At least the functionality works now. I know that we don't have use cases yet, but now we will see the breadcrumbs.